From 294f98e6cddc7c42da02def6ad7192813b69a092 Mon Sep 17 00:00:00 2001
From: Aitor sanmartin Ferreira <aitor.sanmartin11@gmail.com>
Date: Thu, 13 Sep 2018 22:10:55 +0200
Subject: [PATCH] patch asd

---
 Main.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Main.cpp b/Main.cpp
index 2104e75..30e8743 100644
--- a/Main.cpp
+++ b/Main.cpp
@@ -21,6 +21,12 @@ public:
 };
 
 
+class concurrentTimer
+{
+  public:
+   councurrentTimer(){}
+
+};
 
 class child:public parent{
     
-- 
2.13.6 (Apple Git-96)

